### ISSUE TITLE
Test grid points for evaluating IRF interpolation

### ISSUE THREAD

**Issue by moralejo** [cta-observatory/lst-sim-config#49]
In order to test the IRF interpolation algorithms, we need some additional pointings which we can use as "targets" for the interpolation of the standard test MC grid. By comparing the interpolated IRFs, and those computed directly with these additional test pointings, we can assess the precision of the  interpolation.

I suggest the two orange points below (red is the Crab path):
![image](https://user-images.githubusercontent.com/14164282/179554759-af3485fb-d09f-4d97-b029-304c3cc3c558.png)

They have cos($\theta$) = 0.916154  (i.e. $\theta$ = 23.63 deg),  equal to the mean of  the cos($\theta$) of the pointings above and below  (which correspond to 10.0  and 32.059 degrees), and sin($\delta$) chosen to match the Crab path, i.e. 0.76703 and 0.80839.  In one of them the interpolation would nearly be a 1-D interpolation, since it is aligned with two grid points (but that may actually be a good test).

These pointings have the advantage that we can also use them for other tests involving real Crab data.

Of the four actual pointings in (ZD, Az) (two for each sin $\delta$) we can choose those which are closer to the Crab path:

```
zen(deg)  az(deg)  cone_min(deg) cone_max(deg) max imp(m) Emin(TeV) Emax(TeV)
 23.630   259.265     0.3999       0.4001       764.1      0.006     62.237
 23.630   100.758     0.3999       0.4001       764.1      0.006     62.237
```

As usual, Azimuth is astronomical azimuth, measured from geographic north clockwise (i.e. N-E-S-W).

Please comment if this seems reasonable (particularly @jsitarek @chaimain @RuneDominik  @rlopezcoto  @Voutsi @vuillaut @SeiyaNozaki @morcuended)

**Comment by vuillaut** [comment]
Are new pointings nodes really necessary?
We have  for example the following: `node_theta_14.984_az_175.158_` and `node_theta_43.197_az_175.158_` that should interpolate exactly to `node_theta_32.059_az_175.158_`

**Comment by chaimain** [comment]
> Are new pointings nodes really necessary?
> We have  for example the following: `node_theta_14.984_az_175.158_` and `node_theta_43.197_az_175.158_` that should interpolate exactly to `node_theta_32.059_az_175.158_` 


Yes, it will help us to perform the tests to check the efficiency of the interpolation with finer grid widths to check the dependence of grid width to interpolations (especially quantile interpolation for the pdf based IRFs) and therefore to either justify the current grid widths or propose finer ones.

**Comment by jsitarek** [comment]
the proposal of those extra test pointings looks good to me.
Do you need the full statistics as for the other nodes, or do you think the number of events can be reduced for faster generation?

**Comment by RuneDominik** [comment]
Having dedicated test targets would be quite helpful. To add to the discussion, maybe we'd also want to have a high-zen target for some extensive tests. Additionally, since extrapolation will be needed, an extrapolation target might be desirable (maybe at the vertex of the crab path?).

**Comment by vuillaut** [comment]
> the proposal of those extra test pointings looks good to me. Do you need the full statistics as for the other nodes, or do you think the number of events can be reduced for faster generation?

I  suppose you will want to produce the IRFs at these nodes and compare the calculated IRFs with the interpolated ones, right?
If so, the same statistics is required for fair comparison imo.

**Comment by jsitarek** [comment]
> Having dedicated test targets would be quite helpful. To add to the discussion, maybe we'd also want to have a high-zen target for some extensive tests. Additionally, since extrapolation will be needed, an extrapolation target might be desirable (maybe at the vertex of the crab path?).

I think you can also test the extrapolation with those nodes - just use input nodes that cause the new node to be slightly outside the interpolation triangle (seems easy looking at the plot of Abelardo)

**Comment by jsitarek** [comment]
> > the proposal of those extra test pointings looks good to me. Do you need the full statistics as for the other nodes, or do you think the number of events can be reduced for faster generation?
> 
> I suppose you will want to produce the IRFs at these nodes and compare the calculated IRFs with the interpolated ones, right? If so, the same statistics is required for fair comparison imo.

This was more or less my line of thought - now that we used the original grid for IRFs we have better feeling if the original statistics are fine, or if they should be increased/reduced.

**Comment by moralejo** [comment]
I did not mention it explicitly, but these two nodes will also allow us to use them for the "LST1 performance project", without interpolation, to reduce the systematics from the "IRF vs. data pointing mismatches".

**Comment by moralejo** [comment]
> Having dedicated test targets would be quite helpful. To add to the discussion, maybe we'd also want to have a high-zen target for some extensive tests. Additionally, since extrapolation will be needed, an extrapolation target might be desirable (maybe at the vertex of the crab path?).

I think extrapolation can be well tested with those two points, and compared with interpolation as well as with "true" IRF.
The large zenith is indeed important as well, but I'd like to see the full study of the interpolation precision at low zenith, and with everything we learn from it move on to the higher zenith study.

**Comment by moralejo** [comment]
@Voutsi, can these two nodes be placed in the queue of productions?

**Comment by Voutsi** [comment]
@moralejo sure, I will produce them just after finishing the #44

**Comment by moralejo** [comment]
> @moralejo sure, I will produce them just after finishing the #44

Thanks!!

**Comment by Voutsi** [comment]
Hello, please find the simtel files at:

/home/georgios.voutsinas/ws/AllSky/TestDataset/sim_telarray/node_theta_23.630_az_259.265_/output_v1.4

/home/georgios.voutsinas/ws/AllSky/TestDataset/sim_telarray/node_theta_23.630_az_100.758_/output_v1.4

**Comment by moralejo** [comment]
Awesome, thanks @Voutsi!
@vuillaut @garciagenrique, can you please include these in the test sample, and produce the standard DL2 files using the same "Crab RFs" that we used for the analysis of the standard Crab sample. We need those to have more accurate IRFs for the Crab analysis.

**Comment by Voutsi** [comment]
Hi @jsitarek , the production of the 4 additional nodes at 37 & 47 Zd angle has finished, you can still find them at the same old place

**Comment by jsitarek** [comment]
Thanks a lot @Voutsi ! I started processing the files
